tests/integration_test.py

```diff
@@ -246,30 +247,80 @@ def runTest(self):
         self.assertFalse(inspect_data['VolumesRW'][mount_dest])
 
 
-@unittest.skipIf(NOT_ON_HOST, 'Tests running inside a container; no syslog')
```

Comment:
Yep, I think it is. I ran this test on it's own. I did also run make integration-test but get some time out errors which I believe aren't due to this test. Did you see the commit that goes with this code change? It describes the why of doing this. By using `json-file` as the use case we can run this without needing to be on a machine with syslog installed.